storybook/package.json

```diff
@@ -23,7 +23,10 @@
     "eject": "react-scripts eject",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "test-storybook": "test-storybook"
+    "test-storybook": "test-storybook -i",
+    "storybook-static": "npx http-server storybook-static --port 6006 --silent",
+    "test-storybook:ci": "concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"npm run build-storybook && npm run storybook-static\" \"npx wait-on tcp:6006 && npm run test-storybook\"",
+    "test-storybook-modified:ci": "concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"npm run build-storybook && npm run storybook-static\" \"npx wait-on tcp:6006 && npm run test-storybook\""
```

Comment:
Hard to read. Can we split it up or put it into a bash script? Also, IMO long flags are preferable in scripts for readability (e.g. use `--kill-others` instead of `-k`).
If you don't want an extra script, consider using `'` instead of `"` for CLI args, because it doesn't need to be escaped.

Reply:
It already has been split up some. This follows the recipe given by Storybook. We can also simplify it further if it's necessary.

Reply:
`concurrently`, the package being used here, recommends using double quotes in their docs. Though they don't specify I'd wager it's for cross platform compatibility between windows/mac/linux. I don't think we should change that here especially if it's being offered as an example. If you would really prefer long flags I can swap those out though. Let me know.

Reply:
If it follows storybook docs it should be fine.